3_predictions/src/munge_predictions.R

```diff
@@ -0,0 +1,125 @@
+munge_rgcn_output <- function(in_ind, out_ind) {
+  # put XJ's predictions in folder 3_predictions/in and read in
+  dat_all <- npyLoad(sc_retrieve(in_ind))
+
+  # transform and munge npy data
+  munge_npy <- function(dat) {
+    tdat <- as.data.frame(t(dat))
+    names(tdat) <- c('2005', '2006', '2007', '2008', '2009', '2010', '2011', '2012',
+                     '2013', '2014', '2015', '2016', '2017', '2018', '2019', '2020',
+                     '2021', '2022', '2023', '2024', '2025', '2026', '2027', '2028',
+                     '2030', '2031', '2032', '2033', '2034', '2035', '2036', '2037',
+                     '2038', '2039', '2040', '2041', '2044', '2045', '2046', '2047',
+                     '2048', '4182')
+    tdat$date <- seq.Date(from = as.Date('2004-10-01'), to = as.Date('2016-09-30'), by = 1)
+    
+    dat_long <- tidyr::gather(tdat, key = 'seg_id_nat', value = 'rgcn_temp_c', -date)
```

Comment:
Agreed, this all makes me very uncomfortable :) I hard coded these because dates nor seg_ids were in the file Xiaowei gave us, and passed along these values. For now I'm comfortable leaving it in, as I don't see another great option since I don't know exactly how he is generating seg order.